lib/dslash_mdw_fused.in.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <quda_internal.h>
+#include <color_spinor_field.h>
+#include <gauge_field.h>
+#include <clover_field.h>
+#include <worker.h>
+#include <domain_wall_helper.h>
+#include <fast_intdiv.h>
+
+// Tensor core functions for Mobius DWF
+namespace quda
+{
+
+  namespace mobius_tensor_core
+  {
+
+    template <int Ls>
+    void apply_fused_dslash_impl(ColorSpinorField &out, const ColorSpinorField &in, const GaugeField &U,
+                                 ColorSpinorField &y, const ColorSpinorField &x, double m_f, double m_5,
+                                 const Complex *b_5, const Complex *c_5, bool dagger, int parity, int shift[4],
+                                 int halo_shift[4], MdwfFusedDslashType type);
+
+    template <int...> struct IntList {
+    };
+
+    void inline apply_fused_dslash_list(ColorSpinorField &, const ColorSpinorField &in, const GaugeField &,
+                                        ColorSpinorField &, const ColorSpinorField &, double, double, const Complex *,
+                                        const Complex *, bool, int, int[4], int[4], MdwfFusedDslashType, IntList<>)
+    {
+      errorQuda("Ls = %d has not been instantiated", in.X(4));
+    }
+
+    template <int Ls, int... N>
+    void apply_fused_dslash_list(ColorSpinorField &out, const ColorSpinorField &in, const GaugeField &U,
+                                 ColorSpinorField &y, const ColorSpinorField &x, double m_f, double m_5,
+                                 const Complex *b_5, const Complex *c_5, bool dagger, int parity, int shift[4],
+                                 int halo_shift[4], MdwfFusedDslashType type, IntList<Ls, N...>)
+    {
+      if (in.X(4) == Ls) {
+        apply_fused_dslash_impl<Ls>(out, in, U, y, x, m_f, m_5, b_5, c_5, dagger, parity, shift, halo_shift, type);
+      } else {
+        apply_fused_dslash_list(out, in, U, y, x, m_f, m_5, b_5, c_5, dagger, parity, shift, halo_shift, type,
+                                IntList<N...>());
+      }
+    }
+
+    void inline apply_fused_dslash(ColorSpinorField &out, const ColorSpinorField &in, const GaugeField &U,
+                                   ColorSpinorField &y, const ColorSpinorField &x, double m_f, double m_5,
+                                   const Complex *b_5, const Complex *c_5, bool dagger, int parity, int shift[4],
+                                   int halo_shift[4], MdwfFusedDslashType type)
+    {
+      // clang-format off
```

Comment:
I guess clang-format goes crazy with `@...@` ?

Reply:
Yeah there does not seem to have an option to stop it from formatting `@XXX@` to `@XXX @`.

Reply:
Do we need the `@ONLY` in the `configure_file` call?

Reply:
Well, I guess it is the cleaner solution to use it and tell clang-format to skip that.

Reply:
With the current code there is no need, but just put it there for future safety.